ONSdigital/dp-api-tests
# Add missing dataset API tests

### What

Extend test coverage for missing api endpoints

### How to review

Check changes are logical and make sense. Run acceptance tests by doing the following:

1) Run `dp-auth-api-stub` on `cmd-develop` with `make debug`
2) Run `dp-dataset-api` on `feature/import-task-validation` with `make acceptance-publishing`
3) Run acceptance tests in `dp-api-tests` on `feature/extend-missing-dataset-tests` with `cd publishing/datasetAPI/; go test ./...` 


### Who can review

Anyone
